index.html

```diff
@@ -940,19 +945,37 @@
-mechanism that is not sufficient to support [=privacy=], they do complement self-determination and
-help improve accountability. Such rights should include:
+[=People=] retain certain rights over data about themselves, and these rights should be facilitated by their [=user agent=] and the [=parties=]. While data rights alone are not sufficient to support [=privacy=], they do support self-determination and help improve accountability. Such rights include:
 
 * The right to <em>access</em> [=data=] about oneself.
-* The right to <em>erase</em> [=data=] about oneself, with or without terminating the service (though
-  what [=data=] can be erased may differ between those two cases).
-* The right to <em>port</em> information that one has stored with a given [=party=] in such a way
-  that it can easily be reused or transferred elsewhere.
+
+  This right includes being able to discover data about oneself (no secret databases) and to review what information has been collected or inferred.
+
+* The right to <em>erase</em> [=data=] about oneself, sometimes described as a <em>right to be forgotten</em>. 
+
+  The right to erase applies whether or not terminating use of a service altogether, though what [=data=] can be erased may differ between those two cases. On the Web, [=people=] may wish to erase data on their device, on a server, or both, and the distinctions may not always be clear.
+
+* The right to <em>port</em> [=data=], including data one has stored with a [=party=], so it can easily be reused or transferred elsewhere.
+
+  Portability is needed to realize the ability for [=people=] to make choices about services with different data practices. Standards for interoperability are essential for effective re-use.
+
 * The right to <em>correct</em> [=data=] about oneself, to ensure that one's [=identity=] is
   properly reflected in a system.
 
+* The right to <em>be free from automated decision-making</em> based on data about oneself.
+
+  For some kinds of decision-making with substantial consequences, there is a privacy interest in being able to exclude oneself from automated profiling.
+
+  <aside class="issue">
+    Should we settle on examples of the kind of automated decision-making that is impactful enough that one can opt out of it? Are there direct ways to apply this to the Web generally?
+  </aside>  
+
+* The right to <em>object, withdraw consent and restrict use</em> of data about oneself.
+
+  [=People=] may change their decisions about consent, or may object to subsequent uses of data about themselves. Retaining rights requires ongoing control, not just at the time of collection.
+
+The legal rights of [=people=] as data subjects are found within the OECD Privacy Principles [[OECD-Guidelines]], [[Records-Computers-Rights]] and the [[GDPR]], among other places. These participatory rights by people over data about themselves are inherent to [=autonomy=]. 
```

Comment:
To avoid suggesting that these sources authoritatively define all of our rights:
```suggestion
The OECD Privacy Principles [[OECD-Guidelines]], [[Records-Computers-Rights]], and the [[GDPR]], among other places, include many of the rights [=people=] have as data subjects. These participatory rights by people over data about themselves are inherent to [=autonomy=]. 
```